edx_api/email_settings_test.py

```diff
@@ -0,0 +1,38 @@
+"""test for change email settings"""
+import requests_mock
+from unittest import TestCase
+from urllib.parse import urljoin
+
+from edx_api.client import EdxApi
+from edx_api.email_settings import ChangeEmailSettings
+
+
+class TestChangeEmailSettings(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        edx_base_url = 'http://edx.example.com'
+        cls.api_url = urljoin(edx_base_url, ChangeEmailSettings.api_url)
+        cls.client = EdxApi({'access_token': 'foobar'}, cls.api_url)
+        cls.email_settings = cls.client.email_settings
+        cls.json = {"success": "true"}
+        cls.coure_id = "course_id"
```

Comment:
cls.coure_id = "course_id"
```suggestion
cls.course_id = "course_id"
```